alvr/client_openxr/src/interaction.rs

```diff
@@ -136,6 +136,7 @@ impl InteractionContext {
         let controllers_profile_path = match platform {
             p if p.is_quest() => QUEST_CONTROLLER_PROFILE_PATH, // todo: create new controller profile for quest pro and 3
             Platform::PicoNeo3 => PICO_NEO3_CONTROLLER_PROFILE_PATH,
+            Platform::Pico4Ultra => PICO4S_CONTROLLER_PROFILE_PATH,
```

Comment:
Iirc we fail to properly distinguish between the pico 4 ultra or the pico 4 enterprise, so either that has to be fixed or we have to be sure that the button sets match between those.

Reply:
`Build.PRODUCT`? IDK what Pico 4 Ultra has, but Pico 4 Enterprise has `Pico 4 Enterprise` and Pico 4 Pro has `Pico 4 Pro` values.

Reply:
@The-personified-devil Pico 4 Ultra has different device name than Pico 4. The issue was between pico 4 and 4 enterprise or 4 pro eye, and thanks @curoviyxru for suggesting Build.PRODUCT.

Reply:
@zmerp which is also exactly what I said lol